x-pack/plugins/maps/common/elasticsearch_util/elasticsearch_geo_utils.js

```diff
@@ -280,33 +280,16 @@ export function makeESBbox({ maxLat, maxLon, minLat, minLon }) {
   return esBbox;
 }
 
-function createGeoBoundBoxFilter({ maxLat, maxLon, minLat, minLon }, geoFieldName) {
-  const boundingBox = makeESBbox({ maxLat, maxLon, minLat, minLon });
-  return {
-    geo_bounding_box: {
-      [geoFieldName]: boundingBox,
-    },
-  };
-}
-
-export function createExtentFilter(mapExtent, geoFieldName, geoFieldType) {
-  ensureGeoField(geoFieldType);
-
+export function createExtentFilter(mapExtent, geoFieldName) {
   // Extent filters are used to dynamically filter data for the current map view port.
```

Comment:
This entire comment block can be removed now. It was put in place to describe why geo_point was using geo_bounding_box query instead of a geo_shape compatible query https://github.com/elastic/kibana/pull/62966. Now that geo_shape supports geo_bounding_box, there is compatibility across geo_point and geo_shape so the explanation is no longer needed.